worldedit-bukkit/adapters/adapter-1_17_1/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_17_R1_2/PaperweightGetBlocks.java

```diff
@@ -670,12 +673,29 @@ public synchronized <T extends Future<T>> T call(IChunkSet set, Runnable finaliz
                                 if (entity != null) {
                                     final net.minecraft.nbt.CompoundTag tag = (net.minecraft.nbt.CompoundTag) adapter.fromNative(
                                             nativeTag);
+                                    if (entityTagMap.containsKey("Leash")) {
+                                        var leashTag = (net.minecraft.nbt.CompoundTag) adapter.fromNative(
```

Comment:
Not sure if we want to be using var instead of explicit given the rest of the project is explicit? @IntellectualSites/fastasyncworldedit-team

Reply:
I can change it but i prefer more the new features from java 17 on wards

Reply:
It would be the first occurrence of local variable type inference indeed. But as there is a cast on the RHS and we need to fully qualify the type otherwise, `var` might actually help keeping the code readable here. So, I'm fine with it in this case.

Reply:
Can we maybe prefer on spots/lines without castings to use var to make in future the code more readable ?